politeiawww/api/www/v1/v1.go

```diff
@@ -510,6 +513,7 @@ type ProposalRecord struct {
 	LinkTo              string      `json:"linkto,omitempty"`              // Token of linked parent proposal
 	LinkBy              int64       `json:"linkby,omitempty"`              // UNIX timestamp of RFP deadline
 	LinkedFrom          []string    `json:"linkedfrom,omitempty"`          // Tokens of public props that have linked to this this prop
+	Category            string      `json:"category,omitempty"`            // Proposal category
```

Comment:
I don't think we want to include this as part of a `ProposalRecord`.  This is part of our new strategy of pushing all metadata into plugins and the client.  The `ProposalMetadata` will be returned as part of the `ProposalRecord.Metadata` array and its up to the client to decide what should be done with it.